src/Registration/Common/MutualInformation/Code.cxx

```diff
@@ -58,7 +60,18 @@ main(int itkNotUsed(argc), char * itkNotUsed(argv)[])
   WriterType::Pointer movingWriter = WriterType::New();
   movingWriter->SetFileName("moving.png");
   movingWriter->SetInput(movingImage);
-  movingWriter->Update();
+  movingWriter->Update();*/
+
+  using ReaderType = itk::ImageFileReader<ImageType>;
+  ReaderType::Pointer fixedReader = ReaderType::New();
+  fixedReader->SetFileName("fixed.png");
```

Comment:
If I'm not mistake the input/output filenames can be specified in the `CMakeLists.txt` files to some general argument names. Cannot remember if that applies to C++ as well, but it certainly does to Python. That allows to only change the filenames at a single place when necessary.

Reply:
Updated to rely on filename input in `CMakeLists.txt`.

Reply:
:+1: Thanks for having taken care of that as well.

Reply:
Not sure if I see the change :thinking:.

Reply:
See [lines 42-55 and 261](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/198/files#diff-09d60ecdee331c3f36436c199c007027f6876f7e9e0d498f24bc448ff2316151R44) in the updated [Code.cxx](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/198/files#diff-09d60ecdee331c3f36436c199c007027f6876f7e9e0d498f24bc448ff2316151)

Reply:
You are right concerning the `MutualInformationAffine` example; I meant the `MutualInformation` example.

Reply:
My apologies, I've now updated the [`MutualInformation`](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/198/files#diff-542b537435c3b720044a9e0b278f2c082474075dd682a703cdbdf380cf6bb775) example as well.

Reply:
:+1: Thanks for the effort @tbirdso !